### PR TITLE
Text Not Visible During SignUp/LogIn #515

### PR DESCRIPTION
**Pull request by Hana091285**
All texts visibility has been modified as per the issue request
p1
p2
p3
p4

### CODE REVIEW

File: src/Components/AppStyles.js [hunk 1]
```diff
@@ -20,9 +21,9 @@ export const title = {
 };
 export const borderRadius = 20;
 export const shadow = {
-  shadowColor: darkGreyColor,
+  shadowColor: blackColor,     // change darkgrey color to black color for shadow
```
Comment (dannydaking23) [inline src/Components/AppStyles.js:24]: Shadow color was changed from dark grey to black to increase visibility and allow information to stand out.

File: src/Components/AppStyles.js [hunk 2]
```diff
@@ -142,6 +144,7 @@ export default {
     },
     TextInput: {
       fontSize: regularFontSize,
+      fontWeight: 'bold',  // add new line fontweight for text input for login and signup to make text more visible
```
Comment (dannydaking23) [inline src/Components/AppStyles.js:147]: Font weight added as "bold" for the login/signup page to make the text pop out and be seen easier by the user.

File: src/Components/AppStyles.js [hunk 3]
```diff
@@ -156,6 +159,7 @@ export default {
     justifyContent: 'center',
     backgroundColor: 'white',
     fontSize: regularFontSize,
+    fontWeight: 'bold',  // add new line fontweight for text input mask class for login and signup to make text more visible
```
Comment (dannydaking23) [inline src/Components/AppStyles.js:162]: Font weight added as "bold" once again to increase visibility.

File: src/Components/AppStyles.js [hunk 4]
```diff
@@ -578,6 +582,7 @@ export default {
     },
     TextInput: {
       fontSize: regularFontSize,
+      fontWeight: 'bold',  // add new line text input font weight to make text more visible and darker
```
Comment (dannydaking23) [inline src/Components/AppStyles.js:585]: Font weight for text input set to "bold" to improve visibility.

**Review by dannydaking23 — APPROVED** [review]
All changes both darken shadows and change the font to bold which will greatly increase overall text visibility.

**Comment by edumorlom** [comment]
@Hana091285 I just tried on my iphone and it's not working. Can someone else check that it works on iOS?

**Comment by Hana091285** [comment]
> @Hana091285 I just tried on my iPhone and it's not working. Can someone else check that it works on iOS?

@edumorlom you are absolutely right, I do not know why it's not working for iPhone. I have tried it in my iphone and it's not working but it does work for android devices.

**Pull request merged**